.ci/teardown.sh

```diff
@@ -172,6 +177,9 @@ collect_logs()
 		echo "Kernel Log:"
 		sudo journalctl --no-pager -t kernel
 
+		echo "Runtime Tracing Log:"
+		sudo journalctl --no-pager -t kata-runtime
```

Comment:
here you are getting the kata-runtime log... which is already collected above.
I think you'll need to collect what is inside `/var/log/kata-tests/traces`.
See: https://github.com/kata-containers/tests/blob/master/tracing/tracing-test.sh#L20